server/routes/parserRoutes.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const router = express.Router();
+const { mealTimes, datesLog } = require("../models/Identifier");
+const parseFunction = require("../services/Parser");
+
+router.get('/', function (req, res) {
+  res.send("Parse It!");
+});
+
+//TODO Fulfill endpoint to list Meal Times
+router.get('/mealtimes', function (req, res) {
+  res.json(mealTimes);
+});
+//TOD Fulfill endpoint to list Dates
+router.get('/dates', function (req, res) {
+  res.json(datesLog);
+});
+
+//TODO Parse the query!
+router.post('/parse', function (req, res) {
+  const { str } = req.body;
+  const parsed = parseFunction.parser(str);
+  res.json(parsed);
+});
```

Comment:
Maybe you can break these into controllers. So it's easier to maintain in the future
```suggestion
router.post('/parse', parse);
```